src/components/game/GamePage.js

```diff
@@ -41,6 +41,7 @@ export default function GamePage() {
     }
     globalContext.socket.on('roomRoles', (data) => {
       console.log('ROOM ROLES: ', data)
+      globalContext.updateScreenshot(undefined, globalContext)
```

Comment:
Not sure why github won't let me comment on non-edited code (+1 for Bitbucket), but for your states I would try to match the initial default value with the value you are setting. (i.e., `const [blueTeamWord, setBlueTeamWord] = useState('')`). It's a bit more readable and you'll know what primitive/object you'll need to set. If your initial state needs to be `undefined` (which I don't think it should be in most cases), I believe you could also use `const [state, setState] = useState()`.